File: NeetCode/Blind75/stack/mySolution.cs
public class Solution {
    public bool IsValid(string s) {
        // Console.WriteLine($"{s}");
        Stack<char> stck = new Stack<char>();
        foreach (char ch in s)
        {
            if (ch == '(' || ch == '{' || ch == '[') 
            {
                stck.Push(ch);
            }
            else if (ch == ')' || ch == '}' || ch == ']')
            {
                if (stck.Count() == 0 ||
                    (stck.Peek() != '{' && ch=='}') || 
                    (stck.Peek() != '(' && ch==')') ||
                    (stck.Peek() != '[' && ch==']'))
                    {
                        return false;
                    }
                stck.Pop();
            }
        }
        if (stck.Count() == 0)  
            return true;
        return false;
    }
}

File: LeetCode/Medium/Stacks/150. Evaluate Reverse Polish Notation/mySolution.py
class Solution:
    def evalRPN(self, tokens: List[str]) -> int:
        stack = []
        for ch in tokens:
            if ch not in "+*-/":
                stack.append(int(ch))
            else:
                x, y = stack.pop(), stack.pop()
                if ch=='+':
                    stack.append(y+x)
                elif ch=='-':
                    stack.append(y-x)
                elif ch=='*':
                    stack.append(y*x)
                elif ch=='/':
                    stack.append(int(float(y)/x))
        return stack.pop()